pleskdistup/actions/emails.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import json
+import subprocess
+
+from pleskdistup.common import action, log, util
+
+
+class IncreaseDovecotDHParameters(action.ActiveAction):
```

Comment:
For the sake of generality I propose to call that like `SetMinDovecotDhParamSize` and make `dhparam_size` configurable.